### PR TITLE
Write tests for validation utils

### PR DESCRIPTION
**Pull request by MarionJHolloway**
# Description
I've refactored the testing for validation so that the general functionality is tested in as part of the utils script and the job only tests that the job runs. This will make testing subsequent validation jobs easier and will keep the number of tests we need to a minimum.

In a later PR I plan to handle and test the size of dataset calculations.

I have removed the parsing of failures as we will probably not want or need this behaviour generally.

# How to test
Unit tests passing
Run in branch here: https://eu-west-2.console.aws.amazon.com/states/home?region=eu-west-2#/v2/executions/details/arn:aws:states:eu-west-2:344210435447:execution:write-tests-for-validation-uti-Ind-CQC-Filled-Post-Estimates-Pipeline:1644e368-84de-47fb-9a0d-2a622c9a372e

# Developer checklist
- [ ] Unit test
- [ ] Linked to Trello ticket
- [ ] Documentation up to date

### CODE REVIEW

File: tests/test_file_data.py [hunk 1]
```diff
@@ -36,6 +36,7 @@
 from utils.cqc_ratings_utils.cqc_ratings_values import (
     CQCRatingsValues,
 )
+from utils.validation.validation_rule_names import RuleNames as RuleName
```
Comment (garypolzin) [inline tests/test_file_data.py:39]: is this required?

File: utils/validation/validation_utils.py [hunk 2]
```diff
@@ -1,3 +1,8 @@
+import os
+import sys
```
Comment (garypolzin) [inline utils/validation/validation_utils.py:2]: is sys required?

**Review by garypolzin — APPROVED** [review]
couple of imports I'm not sure are required but otherwise all looks good!

File: tests/test_file_data.py (hunk 1 above)
Comment (MarionJHolloway) [reply tests/test_file_data.py:39]: Yes

File: utils/validation/validation_utils.py (hunk 2 above)
Comment (MarionJHolloway) [reply utils/validation/validation_utils.py]: Nope, removed thanks!

**Pull request merged**